getReducedProblem.m

```diff
@@ -0,0 +1,144 @@
+function [newParameters,newProperties,newObjectiveFunction,newOptions] = getReducedProblem(parameters,properties,objectiveFunction,options)
```

Comment:
Please document this function and those below

Reply:
General comment on this function:

This function is not used in the current code version. Jan started working on it in #121 to restructure the way how fixed parameters are currently handled.
However, I think this PR fixes enough issues already and we  should not include these files right now, even if we agreed on using them later. I suggest leaving the current implementation for fixed parameters for this PR.
Any objections to this idea?

Reply:
I agree.
(I just want to mention here that most of the functions do not support fixed parameters. This should be addressed soon.)

Reply:
Important to mention that, yes.
To the best of my knowledge: getMultiStarts and getParameterProfiles should support this feature atm.

Reply:
profiles did not work for me.

Reply:
However, the details do not matter. Let's separate the issue / fixes.